diff -ur apache-log4cxx-0.10.0.orig/src/test/cpp/util/filter.h apache-log4cxx-0.10.0/src/test/cpp/util/filter.h
--- apache-log4cxx-0.10.0.orig/src/test/cpp/util/filter.h	2008-04-01 00:33:32.000000000 +0200
+++ apache-log4cxx-0.10.0/src/test/cpp/util/filter.h	2014-06-02 15:09:21.144196720 +0200
@@ -31,7 +31,7 @@
 #define BASIC_PAT "\\[0x[0-9A-F]*] (FATAL|ERROR|WARN|INFO|DEBUG)"
 #define ISO8601_PAT "[0-9]\\{4\\}-[0-9]\\{2\\}-[0-9]\\{2\\} [0-9]\\{2\\}:[0-9]\\{2\\}:[0-9]\\{2\\},[0-9]\\{3\\}"
 #define ABSOLUTE_DATE_AND_TIME_PAT \
-        "[0-9]\\{1,2\\} .* 200[0-9] [0-9]\\{2\\}:[0-9]\\{2\\}:[0-9]\\{2\\},[0-9]\\{3\\}"
+        "[0-9]\\{1,2\\} .* 20[0-9][0-9] [0-9]\\{2\\}:[0-9]\\{2\\}:[0-9]\\{2\\},[0-9]\\{3\\}"
 #define ABSOLUTE_TIME_PAT "[0-2][0-9]:[0-9][0-9]:[0-9][0-9],[0-9][0-9][0-9]"
 #define RELATIVE_TIME_PAT "^[0-9]+"
 
